content/articles/2018-07-roxygen2-6-1-0.Rmarkdown

```diff
@@ -0,0 +1,79 @@
+---
+title: roxygen2 6.1.0
+date: 2018-07-30
+slug: roxygen2-6-1-0
+author: Mara Averick
+categories: [package]
+description: >
+    roxygen2 6.1.0 is now on CRAN!
+photo:
+  url: https://unsplash.com/photos/3KEJMfoHoMY
+  author: Graeme Nicholl
+---
+
+```{r setup, include = FALSE}
+library(roxygen2)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're pleased to announce that [roxygen2](https://CRAN.R-project.org/package=roxygen2) 
+6.1.0 is now on CRAN. The goal of roxygen is to make documenting your code as 
+easy as possible. It allows you to describe your functions in comments next to 
+their definitions, and processes your source code and comments to produce `.Rd` 
+files in the `man/` directory, as well as. roxygen will also create a 
+`NAMESPACE` for you, and will manage the `Collate` field in `DESCRIPTION`.
+
+This release includes several updates to markdown parsing, as well as some new 
+features we highlight below. For a full list of updates and changes, please 
+see the [release notes](https://github.com/klutometis/roxygen/blob/master/NEWS.md#roxygen2-610).
+
+## New features
+
+The `NAMESPACE` roclet now works in two passes - it first generates the 
+`NAMESPACE` containing only import directives because this can be generated 
+without evaluating the code in the package. This alleviates a problem where it 
+was previously possible to get into a state that you could only get out of by 
+carefully editing the `NAMESPACE` by hand.
+
+`roxygenise()` now uses `pkgload::load_all()` instead of a homegrown solution 
```

Comment:
"This is one of the benefits of the devtools conscious uncoupling" + link to that section on devtools readme/homepage